app/routers/profile.py

```diff
@@ -1,15 +1,14 @@
 import io
 
-from fastapi import APIRouter, Depends, File, Request, UploadFile
-from starlette.responses import RedirectResponse
-from starlette.status import HTTP_302_FOUND
-from PIL import Image
-
 from app import config
 from app.database.database import get_db
 from app.database.models import User
 from app.dependencies import MEDIA_PATH, templates
-
+from app.internal.on_this_day_events import get_on_this_day_events
+from fastapi import APIRouter, Depends, File, Request, UploadFile
+from PIL import Image
+from starlette.responses import RedirectResponse
+from starlette.status import HTTP_302_FOUND
```

Comment:
Your import sorting setting isn't configured correctly.

Reply:
can you please helpme to find the right config? or i can just revrt the sorting @Gonzom

Reply:
I can help with PyCharm, as that is what I'm using.
1. Go to File-> Setting (ctrl+alt+s)
2. Editor
3. Code Style
4. Python
5. Go to the Imports tab
6. Make sure the following are marked: 
6.1. Sort import statements
6.2 Sort import statements in "from" imports
6.3 Sort case-insensitively (not sure about this one)

Reply:
Im using vscode, i cant find those settings,
i see only python.sortImports.args that i can pass but its not seems to be the setting you mentioned.
@Gonzom